Assignment/solution_student/css/style.css

```diff
@@ -1,27 +1,231 @@
 /*
-Project Name: HTML Template
-Client: Your Client
-Author: Your Name
-Developer: @poopsplat in Chicago
+Project Name: Relaxr
+Client: The least demanding client in the history of client-facing work.
+Author: Someone B. Clever
+Developer: elsner.laura@gmail.com in Chicago
 */
 
 /******************************************
-/* BASE STYLES
+/* SETUP
 /*******************************************/
-body {
-  color: black;
-  font-size: 16px;
-  background: white;
+/* Box Model Hack: http://www.paulirish.com/2012/box-sizing-border-box-ftw/ */
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+html {
+  box-sizing: border-box;
+}
+/* Micro clearfix hack: http://nicolasgallagher.com/micro-clearfix-hack/ */
+.clearfix {
+  *zoom: 1;
+}
+.clearfix:before,
+.clearfix:after {
+	mar
+    content: " ";
+    display: table;
+}
+.clearfix:after {
+  clear: both;
 }
 
 /******************************************
-/* LAYOUT
+/* GLOBAL STYLES
 /*******************************************/
 
-/******************************************
-/* PAGES
-/*******************************************/
+/*COLORS
+benefits background color: #f0efef
+yellow: #f9e42e
+dark blue: #033048
+light grey: #606161
+almost black: #121212
+*/
+
+
+body {
+	font-family: "Open Sans", Tahoma, Arial;
+/*	width: 1200px;
+	margin: auto;*/
+
+	/*Just keeping this in here in case we are doing a fixed width for part 2 of the assignment.*/
+}
+
+a {
+	text-decoration: none;
+}
+
+h1 {
+	font-weight: 800;
+	font-size: 24px;
+}
+
+h2 {
+	font-size: 24px;
+	font-weight: 600;
+}
+
+blockquote {
+	font-style: italic;
+	font-weight: 300;
+}
+
+footer {
+	background: #121212;
+}
+
+
 
 /******************************************
 /* SECTIONS
-/*******************************************/
\ No newline at end of file
+/*******************************************/
+
+.hero {
+	color: #fff;
+	padding: 20px;
+	height: 600px;
+	background-color: #033048;
+	background-image: url(http://i.imgur.com/LmUguRt.jpg);
```

Comment:
In this instance, the relative url would be as follows:  
`background-image: url('../images/header_bg.jpg');`  
Because your CSS file, where you're making the reference to the image file, is in `/css`, you need to go back up to the root folder: `../`, and then back into the images folder: `../images`. If you need a refresher, maybe this little explanation will help: https://css-tricks.com/quick-reminder-about-file-paths/